modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts

```diff
@@ -262,6 +266,7 @@ export default {
 	'start-test': 'Start Test',
 	'static': 'Static',
 	'status': 'Status',
+	'status-template': 'STATUS',
```

Comment:
Translation should follow the same key and value text.

So if the key is `status-template` the translation should be `Status Template`

```suggestion
	'status-template': 'Status Template',
```

If you want to use STATUS you should use `i18n.translate('status').toUpperCase()`